.appveyor.yml

```diff
@@ -229,7 +229,10 @@ init:
   - cmd: ssh-keygen -f C:\Users\appveyor\.ssh\id_rsa -N ""
   - sh: ssh-keygen -f ~/.ssh/id_rsa -N ""
   # deploy the datalad installer
-  - appveyor DownloadFile https://raw.githubusercontent.com/datalad/datalad-installer/master/src/datalad_installer.py -FileName datalad_installer.py
+  - sh: "[ -z \"${APPVEYOR_DL_INSTALLER_VERSION}\" ] && export APPVEYOR_DL_INSTALLER_VERSION=master || true"
+  - cmd: "IF NOT DEFINED APPVEYOR_DL_INSTALLER_VERSION set APPVEYOR_DL_INSTALLER_VERSION=master"
+  - sh: "appveyor DownloadFile \"https://raw.githubusercontent.com/datalad/datalad-installer/${APPVEYOR_DL_INSTALLER_VERSION}/src/datalad_installer.py\" -FileName datalad_installer.py"
+  - cmd: "appveyor DownloadFile https://raw.githubusercontent.com/datalad/datalad-installer/%APPVEYOR_DL_INSTALLER_VERSION%/src/datalad_installer.py -FileName datalad_installer.py"
```

Comment:
just an idea: now that installer is released, may be just go for `pip install datalad-installer==0.2.0` and avoid "manual" downloads completely?  (invocation should also be adjusted to just invoke `datalad-installer`)